core/src/nusi/fixed-select.tsx

```diff
@@ -17,9 +17,13 @@ import { get } from 'lodash';
 
 const { Option, OptGroup } = Select;
 const FixedSelect = React.forwardRef((props, ref) => {
-  const options = props.children || get(props, 'options');
+  const { options: propsOptions, ...rest } = props;
```

Comment:
u defined a variable names `propsOptions`, but never use it?

Reply:
Just to pick all the properties in props except the options, and the alias name is used so as not to conflict with options in the following code.

Reply:
Options is a read-only property and cannot delete.